docs/dev/release.rst

```diff
@@ -27,106 +30,98 @@ be "vX.YrcN" (also see `PEP 440 <https://www.python.org/dev/peps/pep-0440/>`_).
 
 Choose your adventure:
 
-* :ref:`release-long`
-* :ref:`release-short`
+* :ref:`release-feature`
+* :ref:`release-bugfix`
+* :ref:`release-old`
 
 
-.. _release-long:
+.. _release-feature:
 
-The long way
-============
+Releasing a minor or major version
+==================================
 
-This way is recommended if you are new to the process or wish to manually run
-some automated steps locally. It takes longer but has a smaller risk factor.
-It also gives you a chance to test things out on a machine that is different
-from the one used for deployment on GitHub Actions.
-
-It is recommended for you to have a clean checkout of the Jdaviz repository
-(not the fork), especially if you also do a lot of development work.
-You can create a clean checkout as follows (requires
-`SSH setup <https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh>`_)::
+The automated release infrastructure has proven to be reliable for a good number
+of releases now, so we've been using this faster version as the default release
+procedure. The longer procedure we previously used is still available in the
+:ref:`release-old` section.
 
-    mkdir jdaviz_for_release
-    cd jdaviz_for_release
-    git clone git@github.com:spacetelescope/jdaviz.git .
-    git fetch origin --tags
+You can do a release from your fork directly without a clean code check-out.
 
 #. Ensure `CI on Actions for main <https://github.com/spacetelescope/jdaviz/actions/workflows/ci_workflows.yml?query=branch%3Amain>`_
    and `RTD build for latest <https://readthedocs.org/projects/jdaviz/builds/>`_
    are passing.
 
+#. Lock down the ``main`` branch of the repository by setting the
+   `branch protection <https://github.com/spacetelescope/jdaviz/settings/branches>`_
+   rule for ``main`` to some high number required to merge, so that more PRs don't
+   get merged while you're releasing.
+
+#. Create a new branch on your fork and make sure you have updated tags too. Note
```

Comment:
I think "fork" would suggest pushing to `origin`, whereas later you have this push to `upstream` directly.

```suggestion
#. Create a new local branch and make sure you have updated tags too. Note
```